docs/codingConventions.md

```diff
@@ -0,0 +1,14 @@
+
+# Coding Conventions
+
+We use a coding standard derived from [TEAMMATES'](https://docs.google.com/document/pub?id=1iAESIXM0zSxEa5OY7dFURam_SgLiSMhPQtU0drQagrs&embedded=true), which is a customised variant of Google's.
+
+We've tweaked a few of the rules for our needs:
+
+- Conventions we don't follow: Google copyright notice, named TODOs
+- Import order matches Eclipse's default (no priority given to Google-specific package names)
+- Such `__METHOD_NAMES__` are allowed
+- Line length is increased to 120
+- `if` and `else` blocks don't need to have braces. This is useful for guard clauses and automatically-generated equals methods.
```

Comment:
Like this? `if (true) { return 1; }`

We can, but we'd have to update all the existing auto-generated `equals` methods. That could probably be a separate issue?